recipes/install-kibana.rb

```diff
@@ -0,0 +1,59 @@
+# Cookbook Name:: mh-opsworks-recipes
+# Recipe:: install-kibana
+
+::Chef::Recipe.send(:include, MhOpsworksRecipes::RecipeHelpers)
+
+include_recipe "mh-opsworks-recipes::update-package-repo"
+install_package('nginx')
+install_package('apache2-utils')
+
+install_nginx_logrotate_customizations
+
+elk_info = get_elk_info
+
+kibana_version = elk_info[:kibana_version]
+kibana_checksum = elk_info[:kibana_checksum]
+download_path = "#{::Chef::Config[:file_cache_path]}/kibana.tar.gz" 
```

Comment:
FWIW, I'm going to use this pattern in another couple recipes to avoid a coupling to s3 I've got.